main.py

```diff
@@ -0,0 +1,69 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+import crud
+import models
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/")
+def root() -> dict:
+    return {"message": "Hello World"}
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(db: Session = Depends(get_db)) -> list[str]:
```

Comment:
no it returns list[Author]
Please test your code. Or I mistake pin a screenshot

Reply:
Fix in all places